config/initializers/filters.rb

```diff
@@ -0,0 +1 @@
+FILTERS = %w[name stars price category].freeze
```

Comment:
Let’s put this back in the controller just so you don’t have to go looking for it.